### DISCUSSION TITLE
MagickRectangle with negative values?

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.NET#1459] [General] [answered]
Hi,

I was wondering if `MagickRectangle` values can be negative.
Since `Percentage` can be negative (according to tests), I understand with & height can be negative.

But it's kinda weird.

Thanks,

Regards.

**Comment by dlemstra** [comment, marked as answer]
Yeah that's a good point. I should add some checks for this. I just pushed a patch to add checks for this.
